optaplanner-examples/src/test/java/org/optaplanner/examples/vehiclerouting/optional/score/VehicleRoutingConstraintProviderTest.java

```diff
@@ -64,7 +64,7 @@ public void vehicleCapacityPenalized() {
 
         constraintVerifier.verifyThat(VehicleRoutingConstraintProvider::vehicleCapacity)
                 .given(vehicleA, customer1, customer2)
-                .penalizesBy(20L);
+                .penalizesLong(20);
```

Comment:
This is the main reason why penalizesBy() can't use overloading for penalizesBy(long).